hooks/post_gen_project.py

```diff
@@ -22,10 +22,17 @@
 
 DEBUG_VALUE = "debug"
 
+create_nextjs_frontend = "{{ cookiecutter.create_nextjs_frontend }}" == "y"
+if not create_nextjs_frontend:
+    shutil.rmtree("frontend")
+
 create_react_frontend = "{{ cookiecutter.create_react_frontend }}" == "y"
 if not create_react_frontend:
-    shutil.rmtree("frontend")
+    shutil.rmtree("frontend_react")
 
+# by default we have frontend folder for nextjs. If we want react then we rename react folder to be the frontend folder
+if create_react_frontend:
+    shutil.move("frontend_react", "frontend")
```

Comment:
If I say "no" to nextjs and "yes" to react, there will only be a "frontend" folder, so I expect this line to fail.

Reply:
It is not. If react app stays in frontend_react folder. So if you say "no" to nextjs previous lines will remove the existing "frontend" folder which was nextjs related. And then if it is "yes" for react, this line will update the folder name for react

Reply:
This is the content of a freshly generated scaf project:
```
ls myproject
backend  docs  frontend  k8s terraform
```
Note, there is no `frontend_react` folder.

Adding just those two lines to the hooks, causes this failure:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/shutil.py", line 816, in move
    os.rename(src, real_dst)
FileNotFoundError: [Errno 2] No such file or directory: 'frontend_react' -> 'frontend/frontend_react'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/tmpdg6wznnn.py", line 30, in <module>
    shutil.move("frontend_react", "frontend")
  File "/usr/local/lib/python3.10/shutil.py", line 836, in move
    copy_function(src, real_dst)
  File "/usr/local/lib/python3.10/shutil.py", line 434, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/local/lib/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'frontend_react'
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
Failed to create project.
```

Reply:
My bad on not pushing the folders. Just pushed the folder changes. For now frontend folder (which supposed to have nextjs in it will be empty and following PR will include it)

Reply:
No need to focus on the changes in "{{cookiecutter.project_slug}}/frontend_react" since it is just renaming of the existing folder name frontend -> frontend_react